sqlglot/transforms.py

```diff
@@ -296,12 +297,16 @@ def unqualify_unnest(expression: exp.Expression) -> exp.Expression:
     return expression
 
 
-def unnest_to_explode(expression: exp.Expression) -> exp.Expression:
+def unnest_to_explode(
+    expression: exp.Expression,
+    unnest_using_arrays_zip: bool = True,
+    support_unnest_func: bool = False,
+) -> exp.Expression:
```

Comment:
Yes, support_unnest_func is only for generalization consideration. In my company's engine `clickzetta`, it supports unnest function. I think it is necessary to leave a parameter in the unnest function of non-lateral view.

Reply:
In addition, we look forward to improving the code of clickzetta's dialect. If there is a chance, it will be merged into sqlglot later, which will be very exciting =)

Reply:
Let's remove this parameter for now, it's currently dead code which doesn't serve an immediate purpose. If another dialect (e.g. Clickzetta) needs it at some point, we can reconsider it.